exercises/phone-number/phone_number_test.go

```diff
@@ -20,19 +22,24 @@ var numberTests = []testCase{
 }
 
 func TestNumber(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
 	for _, test := range numberTests {
 		actual, actualErr := Number(test.input)
-		if actual != test.expected {
-			t.Errorf("Number(%s): expected [%s], actual: [%s]", test.input, test.expected, actual)
-		}
-		// if we expect an error and there isn't one
-		if test.expectErr && actualErr == nil {
+		if !test.expectErr {
+			if actualErr != nil {
+				// if we don't expect an error and there is one
+				var _ error = actualErr
+				t.Fatalf("Number(%s): expected no error, but error is: %s", test.input, actualErr)
+			}
+			if actual != test.expected {
+				t.Fatalf("Number(%s): expected [%s], actual: [%s]", test.input, test.expected, actual)
+			}
+		} else if actualErr == nil {
+			// if we expect an error and there isn't one
 			t.Errorf("Number(%s): expected an error, but error is nil", test.input)
```

Comment:
@kytrinyx I have seen `Fatalf` in the other Exercises' test case, so thought of converting `Errorf` to `Fatalf`
Let me know if you want this change to `Errorf`